iis/driver.go

```diff
@@ -40,7 +40,7 @@ const (
 
 	// pluginVersion allows the client to identify and use newer versions of
 	// an installed plugin
-	pluginVersion = "v0.1.1"
+	pluginVersion = "v0.1.2"
```

Comment:
Not related to this PR, but we should release the version under [`releases`](https://github.com/Roblox/nomad-driver-iis/releases). So that users can download the binary directly from the `releases` page rather than building it from the source.